stylesheet.css

```diff
@@ -960,9 +1103,120 @@ button:hover {
   }
 
   .menu-icon {
-    position: fixed;
     right: 35px;
-    z-index: 100;
+    z-index: 1;
+  }
+
+  #overlay {
+    padding: 60px 16px;
+  }
+
+  #modal {
+    padding: 18px 16px;
+  }
+
+  .overlay-nav img {
+    width: 14px;
+    height: 14px;
+  }
+
+  .overlay-con {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .theary {
+    margin-top: 12px;
+    height: auto;
+  }
+
+  .theary p {
+    font-family: 'Poppins', 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .overlay-list {
+    width: 306px;
+    height: 157px;
+  }
+
+  .info-3 {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 203px;
+    height: 50px;
+    border-bottom: 1px solid #ebecf0;
+    margin-top: 12px;
+  }
+
+  .btn {
+    display: flex;
+    flex-direction: row;
+    margin-top: 25px;
+    gap: 12px;
+  }
+
+  .btn-1 {
+    width: 136px;
+    height: 48px;
+
+    /* Primary/B400 */
+
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    padding: 10px 12px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    /* Body / 3 - Bolder */
+
+    font-family: 'Poppins', 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+
+    /* identical to box height, or 133% */
+
+    /* Primary/B500 */
+
+    color: #396df2;
+  }
+
+  .btn-2 {
+    width: 136px;
+    height: 48px;
+    padding: 10px 12px;
+
+    /* Primary/B400 */
+
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    /* Body / 3 - Bolder */
+
+    font-family: 'Poppins', 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+
+    /* identical to box height, or 133% */
+
+    /* Primary/B500 */
+
+    color: #396df2;
```

Comment:
- [ ] Please make sure that your modal follows the Figma template details you can reduce for example the project image size then the user doesn't need to scroll too long to see the details at the bottom of the modal. :+1: 

- [ ] Please make sure that your modal technology list is visible and follow the design on Figma template because as a developer it's important to pay attention to details :+1: